index.js

```diff
@@ -79,10 +78,7 @@ function PinoColada () {
 
   function formatDate () {
     var date = new Date()
-    var hours = padLeft(date.getHours().toString(), 2, '0')
-    var minutes = padLeft(date.getMinutes().toString(), 2, '0')
-    var seconds = padLeft(date.getSeconds().toString(), 2, '0')
-    var prettyDate = hours + ':' + minutes + ':' + seconds
+    var prettyDate = date.toISOString()
```

Comment:
I find `date.toISOString()` a bit too convoluted to read. If you want timezone information, I'd probably extract it out and add it to hours/minutes/seconds formatting.

The thing I am not entirely sure about, is whether timezone information is really needed? PinoColada is meant to be used as a development pretty printer, so the timezone that shows up is the one that you as the person doing the development is in. How are you using this that more than one timezone is needed?

Reply:
A lot of work I do is while SSH'd into various boxes all over the world, so it's nice to see times in a standard timezone, rather than having to remember what the local time would be in each location. Also, we are using this for our production logs (although maybe we shouldn't be?) 
 which is why dates would be good. Maybe this should be customizeable somehow? For example, there could be a default that does the HH:MM:SS pretty format, and then some env var that can be any date format you like?

Reply:
ahhh gotcha! If you're using this in production logs, it'd definitely make sense to have it in a more comprehensive date format.

I like the idea of having an env var for this. Would you mind tweaking this PR to include it?